### ISSUE TITLE
Response too long with M486

### ISSUE THREAD

**Issue by LittleHobbyShop** [Duet3D/DuetWebControl#252]
I'm not sure if this is expected behaviour or not but I'm getting response too long warning with M486 on a print with 10 objects. Sometimes the console lists them all, sometimes not.

GCODE File attached.

[CFFFP_Pi Camera Back.gcode.txt](https://github.com/Duet3D/DuetWebControl/files/4638709/CFFFP_Pi.Camera.Back.gcode.txt)

**Comment by chrishamm** [comment]
The output should be displayed in any case on the G-code console. Code replies that are longer than three lines are truncated and the message displayed is "See console". Can you post a screenshot if anything else is wrong?

**Comment by LittleHobbyShop** [comment]
Seems I jumped the gun. I thought it was behaving strangely by not listing all objects in the console.  On top of that I misinterpreted the 'Response too long' message. It would appear that RRF parses the object list as they are printed and not right from the start so I was trying M486 before all objects had been picked up. DWC seems to be doing exactly as expected.

Below is from a simulation run.

![image](https://user-images.githubusercontent.com/16748684/82216598-12f8a400-9911-11ea-8085-72e5e87b6fed.png)
